client/my-sites/upgrades/checkout-thank-you/custom-domain-purchase-detail.jsx

```diff
@@ -41,31 +44,20 @@ const CustomDomainPurchaseDetail = ( { selectedSite, hasDomainCredit, translate
 					components: { em: <em /> }
 				}
 			) }
-			buttonText={ translate( 'Manage my domains' ) }
-			href={ `/domains/manage/${ selectedSite.slug }` }
-		/>;
-
-	const renderCustomDomainDetail = () => {
-		if ( hasCustomDomain( selectedSite ) ) {
-			return renderHasCustomDomain();
-		}
-
+			{ ...actionButton }
```

Comment:
I'm thinking it might make sense to update purchase detail to accept child components, and remove the PurchaseButton wrapper. (which only adds a class to button and is unnecessary ).

If this feels out of scope, lets mark this for another janitorial